docs/README.md

```diff
@@ -0,0 +1,112 @@
+# wolfTPM User Manual
```

Comment:
I don't see this content in the generated html output. Will comment where we may need to change in Doxyfile.